knowledge-base/how-to-view-web-session-summary.md

```diff
@@ -0,0 +1,101 @@
+---
+title: How to view Web Session Summary
+description: An article on how to capture and inspect web traffic using Fiddler Everywhere
+type: how-to
+slug: how-to-capture-and-inspect-web-traffic
+tags: capture web traffic, inspect web traffic, web traffic, https, web application
+publish: true
+res_type: kb
+---
+
+## View Sessions Summary
+
+Each __Session__ captured by Fiddler Everywhere has multiple attributes ordered in columns. Use these to identify HTTP related issues, to track and debug your site performance, and to leave custom comments.
+
+## (ID)
+
+An auto-generated identification number for the currently listed sessions.
+
+## Result
+
+Indicates the HTTP status code from the response. A server issues the status codes in response to a client's request made to the server and represented by a three-digit number. The first digit of the status code specifies one of five standard classes of responses.
```

Comment:
1) Consider using "of" instead of "from" in the first sentence.
2) It sounds a bit unclear what "and represented by a three-digit number" refers to. In the first part of the sentence the server takes center place. Consider splitting the two sentences, adding the second part to the next sentence or rephrasing to something like:
"Status codes are issued by the server in response to a client's request and are represented by a three-digit number."